Piwik.Tracker/PiwikTracker.cs

```diff
@@ -397,6 +397,15 @@ public string GetCustomTrackingParameter(string trackingApiParameter)
         }
 
         /// <summary>
+        /// Gets the stored tracking actions, that have been stored for bulkRequest <see cref="EnableBulkTracking"/>, <see cref="DoBulkTrack"/>.
+        /// </summary>
+        /// <returns></returns>
+        public string[] GetStoredTrackingActions()

```

Comment:
Could this be made internal?

Also, If tests are only implemented using the mocked http server, can you confirm these fields could remain private?